inc/utils/content.php

```diff
@@ -564,17 +581,19 @@ function get_categories_array($post){
 
  function odm_echo_extras($postID = "") {
  	 $postID = $postID ? $postID : get_the_ID();
-	 if (function_exists('get_post_meta')) :
-		 $get_author = get_post_meta($postID, 'author', true);
-		 $get_localized_author = get_post_meta($postID, 'author_'.odm_language_manager()->get_current_language(), true);
-	   if ($get_author != '' || $get_localized_author != ''):
-	     $news_source_info = '<span class="lsf">&#xE041;</span> ';
-	     if ($get_localized_author != ''):
-	         $news_source_info .= $get_localized_author.'<br />';
-	     else:
-	         $news_source_info .= $get_author.'<br />';
-	     endif;
-	   endif;
+	 if (function_exists('get_post_meta')):
```

Comment:
@acorbi I am happy to share my ideas for UX as below. 
![screencapture-opendevelopmentcambodia-net-announcements-three-new-natural-protected-areas-established-to-be-under-the-management-and-conservation-of-ministry-of-environment-1474449367166](https://cloud.githubusercontent.com/assets/10953792/18705515/c405d6d4-8018-11e6-8419-6aec2d3aedde.png)